fs/accounting.go

```diff
@@ -129,23 +131,37 @@ func (s *StatsInfo) String() string {
 	}
 	dtRounded := dt - (dt % (time.Second / 10))
 	buf := &bytes.Buffer{}
+	
+	transferVolume, transferSpeed := s.bytes, speed
+	transferVolumeUnit, transferSpeedUnit := "Bytes", "Bytes/s"
+
+	if *statsBps {
+		transferVolume, transferSpeed = s.bytes*8, speed*8
+	  	transferVolumeUnit, transferSpeedUnit = "Bits", "Bits/s"
+	}
+	
 	fmt.Fprintf(buf, `
-Transferred:   %10s (%s)
-Errors:        %10d
-Checks:        %10d
-Transferred:   %10d
-Elapsed time:  %10v
+%-20s: %-10s (%-s)
+%-20s: %-10d
+%-20s: %-10d
+%-20s: %-10d
+%-20s: %-10v
 `,
-		SizeSuffix(s.bytes).Unit("Bytes"), SizeSuffix(speed).Unit("Bytes/s"),
+		"Transferred",
+		SizeSuffix(transferVolume).Unit(transferVolumeUnit), SizeSuffix(transferSpeed).Unit(transferSpeedUnit),
```

Comment:
To be extremely pedantic bit rates are almost always specified in SI units rather than the binary units used by SizeSuffix.  So 1 Mbit/s would normally be 1,000,000 bit/s rather than 1,048,576 bits/s.

I suggest we don't fix this though, instead document it!